game/src/devtools/fifteen_min/mod.rs

```diff
@@ -125,12 +158,20 @@ fn build_panel(ctx: &mut EventCtx, start: &Building, isochrone: &Isochrone) -> P
         ])
         .draw(ctx),
     );
+
     for (amenity, buildings) in isochrone.amenities_reachable.borrow() {
         rows.push(
             Btn::text_fg(format!("{}: {}", amenity, buildings.len())).build(ctx, *amenity, None),
         );
     }
 
+    // Start of toolbar
+    rows.push(Widget::horiz_separator(ctx, 0.3).margin_above(10));
+
+    let about_button = Widget::row(vec![Btn::plaintext("About").build_def(ctx, None)]);
+
+    rows.push(about_button);
```

Comment:
Style nit: unless the widget is referenced in multiple places, giving a name to an intermediate variable doesn't seem helpful, since it's clear this is an about button.

Also, no need to do `Widget::row` unless there's 2 things to lay out horizontally. So you can just do `rows.push(Btn::plaintext("About").build_def(ctx, None));`